lib/sidekiq/form526_backup_submission_process/submit.rb

```diff
@@ -52,6 +52,10 @@ class Submit
           'Form 526 Backup Submission Retries exhausted',
           { job_id:, error_class:, error_message:, timestamp:, form526_submission_id: }
         )
+
+        if Flipper.enabled?(:send_backup_submission_exhaustion_email_notice)
+          ::Form526SubmissionFailureEmailJob.perform_async(form526_submission_id: form526_submission_id)
```

Comment:
this should just be 
`::Form526SubmissionFailureEmailJob.perform_async(form526_submission_id:)`
shouldnt it?